spec/ruby/core/kernel/inspect_spec.rb

```diff
@@ -28,4 +28,34 @@ class << obj
     end
     obj.inspect.should be_kind_of(String)
   end
+
+  ruby_version_is "3.5" do
+    it "calls #instance_variables_to_inspect private method to know which variables to display" do
```

Comment:
Reading this I assumed it had to be a private method, but the implementation tells me it doesn't. What do you think about not mentioning private here and making the method public in the empty override case (see next suggestion)?

```suggestion
    it "calls #instance_variables_to_inspect method to know which variables to display" do
```